device/mellanox/x86_64-mlnx_msn3800-r0/platform.json

```diff
@@ -724,576 +724,448 @@
             "index": "1,1,1,1",
             "lanes": "0,1,2,3",
             "breakout_modes": {
-                "1x100G[50G,40G,25G,10G]": ["etp1"],
-                "1x10G[100G,50G,40G,25G]": ["etp1"],
```

Comment:
Please have 1x10G in 202106 as the fix https://github.com/Azure/sonic-buildimage/pull/9278 is not present in 202106